data/text_data/fr_FR.yaml

```
@@ -1,166 +1,6039 @@
+
+- name: Nayru's Wisdom Text
+  standard: "Vous avez appris la <y<Sagesse de Nayru>> !"
+
+- name: Din's Power Text
+  standard: "Vous avez appris la <y<Puissance de Din>> !"
+
+- name: Faron SotH text
+  standard: "Vous avez appris la <y<mélodie du dragon d'eau>> !\nC'est une des mélodies du <y<Chant du héros>> !"
+
+- name: Eldin SotH text
+  standard: "Vous avez appris la <y<mélodie du dragon de feu>> !\nC'est une des mélodies du <y<Chant du héros>> !"
+
+- name: Lanayru SotH text
+  standard: "Vous avez appris la <y<mélodie du dragon de foudre>> !\nC'est une des mélodies du <y<Chant du héros>> !"
+
+- name: Full SotH text
+  standard: "Vous avez appris la <y<dernière mélodie du Chant du héros>>...\n\n\nVous avez reconstitué le <y<Chant du héros>> !"
+
+- name: Cold Pumpkin Soup text
+  standard: "Vous avez obtenu de la <y<soupe de citrouille froide>> !\nC'est la spécialité de Potron, mais quelqu'un l'a laissé refroidir..."
+
+- name: Five Gratitude crystal counter
+  standard: "Vous avez obtenu <r<cinq>> <y<cristaux de gratitude>> !\nÇa en fait <r<<numeric arg1>>> en tout !"
+
+- name: Single Gratitude crystal counter
+  standard: "Vous avez obtenu <r<un>> <y<cristal de gratitude>> !\nÇa en fait <r<<numeric arg1>>> en tout !"
+
+- name: Skyview Key Text
+  standard: "Vous avez obtenu une petite clé pour le <g<temple de la contemplation>> !"
+
+  # This is an intentional duplicate of the above entry
+- name: Item 200 Text
+  standard: "Vous avez obtenu une petite clé pour le <g<temple de la contemplation>> !"
+
+- name: Item 201 Text
+  standard: "Vous avez obtenu une petite clé pour la <y<raffinerie de Lanelle>> !"
+
+- name: Item 202 Text
+  standard: "Vous avez obtenu une petite clé pour la <b<grande caverne antique>> !"
+
+- name: Item 203 Text
+  standard: "Vous avez obtenu une petite clé pour le <r<sanctuaire ancien>> !"
+
+- name: Item 204 Text
+  standard: "Vous avez obtenu une petite clé pour le <y+<galion des sables>> !"
+
+- name: Item 205 Text
+  standard: "Vous avez obtenu une petite clé pour la <s<tour des cieux>> !"
+
+- name: Item 206 Text
+  standard: "Vous avez obtenu une petite clé pour la <ye<grotte de Lanelle>> !"
+
+- name: Item 207 Text
+  standard: "Vous avez obtenu la carte du <g<temple de la contemplation>> !"
+
+- name: Item 208 Text
+  standard: "Vous avez obtenu la carte du <r+<temple de la terre>> !"
+
+- name: Item 209 Text
+  standard: "Vous avez obtenu la carte de la <y<raffinerie de Lanelle>> !"
+
+- name: Item 210 Text
+  standard: "Vous avez obtenu la carte de la <b<grande caverne antique>> !"
+
+- name: Item 211 Text
+  standard: "Vous avez obtenu la carte du <r<sanctuaire ancien>> !"
+
+- name: Item 212 Text
+  standard: "Vous avez obtenu la carte du <y+<galion des sables>> !"
+
+- name: Item 213 Text
+  standard: "Vous avez obtenu la carte de la <s<tour des cieux>> !"
+
+- name: Can't buy Pouch text
+  standard: "Il vous faut la <y<sacoche d'aventurier>> pour acheter ceci !"
```

```suggestion
- name: Can't buy Pouch text
  standard: "Il vous faut la <y<sacoche d'aventurier>> pour acheter ceci !"
```

Needs newline.

---

Ah yeah, I didn't copy over the text correctly. Should be like this

```suggestion
- name: Can't buy Pouch text
  standard: "Il vous faut la <y<sacoche d'aventurier>>\n pour acheter ceci !"
```